### PR TITLE
Calculate

### PR DESCRIPTION
**Pull request by kostuch76**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @kostuch76

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,25 @@
...
+    if (command == "multiply") {
+        return std::to_string(first * second);
+    }
+    if (command == "divide") {
+        if (second != 0) {
+            return std::to_string(first / second);
+        }
+        else {
+            return "Division by 0";
+        }
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:20]: When you use return inside `if` you don't need `if else` or `else`. Just write `if (sth) { return blabla; }`

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -2,6 +2,25 @@
...
+    }
+    else {
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:22]: When you use return inside `if` you don't need `if else` or `else`. Just write `if (sth) { return blabla; }`

**Review by nauka-programowania-MA — APPROVED** [review]
Please push only your modification. Not all files

**Pull request closed without merging**